index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navmenu">
+        <h3>My Logo</h3>
+        <img src="image/Union.png" alt="" />
+      </nav>
+    </header>
+    <section class="backgroundimg">
```

Comment:
- PLease i noticed in the background image it does a repeating image with makes extra white space on the right side of the web page, please add a CSS property on this section of `background-repeat: no-repeat` `background-size:cover`

##### Your design (Take a look at the painted place, the space should not be there)
![Screenshot from 2022-07-02 16-51-51](https://user-images.githubusercontent.com/58771507/177007548-1f6f577f-dd7b-4e0d-8be8-408f6e1e05e8.png)


##### Figma design

![Screenshot from 2022-07-02 16-53-22](https://user-images.githubusercontent.com/58771507/177007605-dc1bd838-c69d-4b4e-9ee6-b680962b1a9d.png)